config/routes.rb

```diff
@@ -64,12 +65,10 @@
   notify_to :users, controller: "users/notifications"
 
   scope "/users" do
-    get "/:id/profile", to: "users/logix#profile", as: "profile"
```

Comment:
what happens to existing links which are posted in blogs and assignments notes of students? will the fallback or redirect to 404 ?

Reply:
Added a fallback; now the old profile links will redirect to the new dashboard

Reply:
Can you add test for that behavior to avoid accidentally removing that condition